File: JavaScript/Exo2/Exo2.js
let a = parseInt(prompt("Saisir le premier nombre !"));
let b = parseInt(prompt("Saisir la deuxieme nombre !"));

let som, diff, prod, quot;

som = a + b;
diff = a - b;
prod = a * b;
quot = a / b;

console.log(`La somme entre ${a} et ${b} est : ${som}`);
console.log(`La difference entre ${a} et ${b} est : ${diff}`);
console.log(`Le produit entre ${a} et ${b} est : ${prod}`);
console.log(`Le quotient entre ${a} et ${b} est : ${quot}`);

alert(`Resultats :\n
    Somme : ${som} \n
    Difference : ${diff} \n
    Produit : ${prod} \n
    Quotient : ${quot}`);

File: C/Exo2/Exo2.c
#include <stdio.h>

int a, b, N;

void main(){

    printf("Saisir la valeur de a \n");
    scanf("%d", &a);

    printf("Saisir la valeur de b \n");
    scanf("%d", &b);

    printf("Saisir la valeur de N \n");
    scanf("%d", &N);

    int Un = 1;
    int sum = Un;

    for(int i = 0; i < N; i++){
        Un = (a * Un) + b;
        sum += Un;
    }
    printf("U(N) est egale a : %d", Un);
    printf("\n");
    printf("La somme des U(n) allant de 0 a N est : %d", sum);
}